### PR TITLE
[SHELL32] 640x480-ify four dialogs

### PR DESCRIPTION
**Pull request by JoachimHenze**
IDD_DRIVE_PROPERTIES
IDD_DRIVE_TOOLS
IDD_DRIVE_HARDWARE
IDD_FOLDER_CUSTOMIZE

I want to offer others to have an early look on the after state.
I covered all RCs, not longer a DRAFT, but readyToReview now.

Afterwards all 4 dialogs do display fine even with 640x480 screen resolution. Same as their MS role-models do.
[after.webm](https://github.com/reactos/reactos/assets/33393466/b36f31d5-ac95-4194-ba65-ec7537473975)

* I also fixed some accelerator collisions FCIDM_SHVIEW_CUT (see the individual commits), and synced some forgotten syncs for IDM_CUT accel
* I also moved checkboxes further to the left in the checkdisk dialog, as it was done for en-US and some other languages before, but it was forgotten in other languages (AUTOCHECKBOX.*14000 and AUTOCHECKBOX.*14001)

JIRA issue: none

## TODO IN FUTURE PR

- [ ] in a future PR I want to do the same for the end-boss "folder properties" propsheet, but don't want to make the PR too large, therefore the split

### CODE REVIEW

File: dll/win32/shell32/lang/da-DK.rc [hunk 1]
```diff
@@ -1,9 +1,3 @@
-/* PROJECT: ReactOS Shell Library
- * FILE:    dll/win32/shell32/lang/da-DK.rc
- * PURPOSE: Danish resource file
- * UPDATED: 01 May 2009
- */
-
```
Comment (tkreuzer) [inline dll/win32/shell32/lang/da-DK.rc:6]: Why remove this? Shouldn't we eventually add the new style file headers everywhere?

**Review by tkreuzer — COMMENTED** [review]
Edit: just realized that there is a lnk to a video.

File: dll/win32/shell32/lang/cs-CZ.rc [hunk 2]
```diff
@@ -1,8 +1,5 @@
-/* FILE:       dll/win32/shell32/lang/cs-CZ.rc
- * TRANSLATOR: Radek Liska aka Black_Fox (radekliska at gmail dot com)
- * UPDATED:    2015-04-12
- * THANKS TO:  navaraf, who translated major part of this file
- */
+// COPYRIGHT: 2015 Radek Liska (Black_Fox) <radekliska@gmail.com>
+//            navaraf
```
[the remark this answers is not in the document]
Comment (JoachimHenze) [reply dll/win32/shell32/lang/cs-CZ.rc:2]: The file section we decided to strip everywhere due to redundandy.
The rest is closer to the new-style headers where every contributor has a line on its own.
Obvious that my focus here is compacting the existing information without stripping anything of worth, (having easy backports in mind)

File: dll/win32/shell32/lang/da-DK.rc (hunk 1 above)
Comment (binarymaster) [reply dll/win32/shell32/lang/da-DK.rc:6]: This adds unnecessary diff to the patch, please do not touch these. I usually help people fixing the headers when actual translation do come in PRs.

Comment (JoachimHenze) [reply dll/win32/shell32/lang/da-DK.rc:6]: I didn't touch any of those headers that were properly new-styled before, and I never do.
This one wasn't properly new-styled ever. So I am free to improve it a bit without going the whole way yet.
You can tweak those headers to your hearts desire after me in your own PR @binarymaster and I won't interfere with that, but please don't force me to make them longer than before within a PR that I do intend to port back. I do really dislike to merge back only 99% of a PR, or to add additonal stuff on top in a backport, that hasn't been done on master beforehand. So please look away just for a moment here.

File: dll/win32/shell32/lang/fr-FR.rc [hunk 3]
```diff
@@ -48,11 +48,11 @@ BEGIN
     MENUITEM SEPARATOR
     MENUITEM "C&ouper\tCtrl+X", FCIDM_SHVIEW_CUT
     MENUITEM "&Copier\tCtrl+C", FCIDM_SHVIEW_COPY
-    MENUITEM "Co&ller\tCtrl+V", FCIDM_SHVIEW_INSERT
-    MENUITEM "Créer un &lien", FCIDM_SHVIEW_INSERTLINK
+    MENUITEM "Co&ller\tCtrl+V", FCIDM_SHVIEW_INSERT //FIXME: accelerator collision &L
+    MENUITEM "Créer un &lien", FCIDM_SHVIEW_INSERTLINK //FIXME: accelerator collision &L
```
Comment (HBelusca) [inline dll/win32/shell32/lang/fr-FR.rc:52]:
```suggestion
    MENUITEM "Coll&er\tCtrl+V", FCIDM_SHVIEW_INSERT
    MENUITEM "Créer un &lien", FCIDM_SHVIEW_INSERTLINK
```

Comment (HBelusca) [reply dll/win32/shell32/lang/fr-FR.rc:52]: In the next menu below, there's a `MENUITEM "Coller", IDM_INSERT` item. You may also assign an accelerator to the "e" letter.

File: dll/win32/shell32/lang/fr-FR.rc [hunk 4]
```diff
@@ -48,11 +48,11 @@ BEGIN
...
     MENUITEM SEPARATOR
-    MENUITEM "Copier vers le &dossier...", FCIDM_SHVIEW_COPYTO
-    MENUITEM "&Déplacer vers le dossier...", FCIDM_SHVIEW_MOVETO
+    MENUITEM "Copier vers le &dossier...", FCIDM_SHVIEW_COPYTO //FIXME: accelerator collision &D
+    MENUITEM "&Déplacer vers le dossier...", FCIDM_SHVIEW_MOVETO //FIXME: accelerator collision &D
```
Comment (HBelusca) [inline dll/win32/shell32/lang/fr-FR.rc:55]:
```suggestion
    MENUITEM "Co&pier vers le &dossier...", FCIDM_SHVIEW_COPYTO
    MENUITEM "&Déplacer vers le dossier...", FCIDM_SHVIEW_MOVETO
```
it looks like the "p" accelerator wasn't used so far in this menu, and thus could be used there.

File: dll/win32/shell32/lang/en-GB.rc [hunk 5]
```diff
@@ -64,7 +64,7 @@ BEGIN
     POPUP ""
     BEGIN
         MENUITEM SEPARATOR
-        MENUITEM "C&ut", IDM_CUT
```
Comment (HBelusca) [inline dll/win32/shell32/lang/en-GB.rc:67]: Why is the "u" separator not OK?

File: dll/win32/shell32/lang/es-ES.rc [hunk 6]
```diff
@@ -56,11 +56,11 @@ BEGIN
     MENUITEM "&Deshacer\tCtrl+Z", FCIDM_SHVIEW_UNDO
     MENUITEM SEPARATOR
     MENUITEM "Cor&tar\tCtrl+X", FCIDM_SHVIEW_CUT
-    MENUITEM "&Copiar\tCtrl+C", FCIDM_SHVIEW_COPY
+    MENUITEM "&Copiar\tCtrl+C", FCIDM_SHVIEW_COPY //FIXME: accelerator collision &C
     MENUITEM "&Pegar\tCtrl+V", FCIDM_SHVIEW_INSERT
     MENUITEM "Pegar &acceso directo", FCIDM_SHVIEW_INSERTLINK
     MENUITEM SEPARATOR
-    MENUITEM "Copiar a &carpeta...", FCIDM_SHVIEW_COPYTO
+    MENUITEM "Copiar a &carpeta...", FCIDM_SHVIEW_COPYTO //FIXME: accelerator collision &C
```
Comment (HBelusca) [inline dll/win32/shell32/lang/es-ES.rc:63]: Any suggestion for these @julenuri ?

File: dll/win32/shell32/lang/it-IT.rc [hunk 7]
```diff
@@ -2,12 +2,12 @@ LANGUAGE LANG_ITALIAN, SUBLANG_NEUTRAL
...
 BEGIN
-    MENUITEM "&Icone Grandi", FCIDM_SHVIEW_BIGICON
-    MENUITEM "&Icone Piccole", FCIDM_SHVIEW_SMALLICON
+    MENUITEM "&Icone Grandi", FCIDM_SHVIEW_BIGICON //FIXME: accelerator collision &I
+    MENUITEM "&Icone Piccole", FCIDM_SHVIEW_SMALLICON //FIXME: accelerator collision &I
     MENUITEM "&Lista", FCIDM_SHVIEW_LISTVIEW
     MENUITEM "&Dettagli", FCIDM_SHVIEW_REPORTVIEW
     MENUITEM "", -1, MFT_SEPARATOR
-    POPUP "Disponi &Icone", FCIDM_SHVIEW_ARRANGE
+    POPUP "Disponi &Icone", FCIDM_SHVIEW_ARRANGE //FIXME: accelerator collision &I
```
Comment (HBelusca) [inline dll/win32/shell32/lang/it-IT.rc:10]: Cc @reactos/lang-italian guys for suggestions of fixes..

**Review by HBelusca — APPROVED** [review]
Looks OK globally; I've added a suggestion for the accelerators collision in the french resources. I also have some minor questions (see comments).

File: dll/win32/shell32/lang/fr-FR.rc (hunk 3 above)
Comment (JoachimHenze) [reply dll/win32/shell32/lang/fr-FR.rc]: Thank you very much for your accelerator suggestions as a native french speaker with a french Windows at hand. Finally a useful review-comment. I applied all your suggestions in the accelerator context.

File: dll/win32/shell32/lang/fr-FR.rc (hunk 4 above)
Comment (JoachimHenze) [reply dll/win32/shell32/lang/fr-FR.rc]: Thank you very much for your accelerator suggestions as a native french speaker with a french Windows at hand. Finally a useful review-comment. I applied all your suggestions in the accelerator context.

File: dll/win32/shell32/lang/en-GB.rc (hunk 5 above)
Comment (JoachimHenze) [reply dll/win32/shell32/lang/en-GB.rc:67]: en-US was initially introduced as &U and then later corrected to &T in the ReactOS history. The chance that en-GB uses a different accelerator than en-US.rc, and then specifically one that we accidentally used here wrongly in the beginning, is close to 0%. en-US and en-GB accelerators are the same in >= 99% of the cases.
Ftr: I gave the individual justification in all the individual changesets. It is worth to have a look at their description, I committed in babysteps to be able to give that exact level of information in there already.

File: dll/win32/shell32/lang/da-DK.rc (hunk 1 above)
Comment (binarymaster) [reply dll/win32/shell32/lang/da-DK.rc:6]: Sorry, but you clearly were not supposed to touch resource headers within the scope of the 640x480-fication of dialogs.

Comment (JoachimHenze) [reply dll/win32/shell32/lang/da-DK.rc:6]: I really learned something from you today binarymaster. The only way being able to work with someone like you is always doing direct pushes, and omit the PR step entirely. That is the only way to workaround your weird obsession with those headers, and being simply quicker than you.

**Pull request merged**